src/components/Pagination.tsx

```diff
@@ -43,19 +43,19 @@ export const Pagination: React.FC<PaginationProps> = ({
 
       <Button>
         <MonospaceText>
-          <Hide if={pages === -1}>
+          <Hide if={pages === null}>
             Page {page} of {pages}
           </Hide>
-          <Hide if={pages !== -1}>{page}</Hide>
+          <Hide if={pages !== null}>{page}</Hide>
         </MonospaceText>
       </Button>
 
       <Hide if={isLastPage || !hasNextPage}>
         <Button onClick={() => onSelect(page + 1)}>
           <ArrowIcon />
         </Button>
-        <Hide if={pages === -1}>
-          <Button onClick={() => onSelect(pages)}>
+        <Hide if={pages === null}>
```

Comment:
you can leave these wrappers but I'v never seen before people needing a wrapper for this basic operation of showing or not showing stuff

in my experience it's more common and explicit to just do

```tsx
<div>
    {someBoolCondition && <>Anything you want here</>}
</div>
```

Reply:
Ok, I updated little bit the legacy code here

Reply:
Updated